src/main/java/org/fundacionjala/sfdc/pageobjects/common/SFCommonObjects.java

```diff
@@ -77,49 +79,50 @@ public void clickElementOnList(final String elementOnList) {
      */
 
     public boolean isWebElementPresentOnList(final String elementOnList) {
-        return CommonActions.istWebElementPresentOnList(this.elementOnList, elementOnList);
+        System.out.println(this.elementOnList);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)